pkg/pillar/uplinkprober/uplinkprober.go

```diff
@@ -191,7 +191,7 @@ func (p *UplinkProber) StartNIProbing(niConfig types.NetworkInstanceConfig) (
 	initialStatus NIProbeStatus, err error) {
 	p.Lock()
 	defer p.Unlock()
-	if !types.IsSharedPortLabel(niConfig.PortLogicalLabel) {
+	if !types.IsEveDefinedPortLabel(niConfig.PortLogicalLabel) {
```

Comment:
If this will remain the UplinkProber, then checking that the label includes one or more management ports makes sense. But that would be a different check than looking for the well-known labels AFAICT.

Reply:
You can pretty much ignore this change. It is done just to keep everything working and without change until the next PR, where uplinkprober will be removed and portprober will be used instead. Sorry for the confusion, couldn't think of better PR split.